### ISSUE TITLE
Was it okay to remove the "controlPanel" from the view?

### ISSUE THREAD

**Issue by jbphet** [phetsims/john-travoltage#295]
In https://github.com/phetsims/john-travoltage/commit/701e3ce7e13c744171b09cfb9093e43a0ba3cb20 and https://github.com/phetsims/john-travoltage/commit/4e40d822e67d91cb41ed420279f31af5cd6dad76 I removed a control panel that previously contained the reset all button and the sound control toggle button.  I did this because the sound control toggle button was being removed as part of the process to add updated sound to this sim, and it didn't make sense to me to have an invisible control panel with only the reset all button on it.  However, this control panel had some a11y-specific settings, and I'm not confident that removing it won't cause some sort of issue with the a11y behavior.

Assigning to @jessegreenberg as my best guess as to who should look at this, he can reassign as appropriate.

**Comment by jbphet** [comment]
I accidentally removed an additional string, and added it back with the commit above.

**Comment by jessegreenberg** [comment]
That Node did provide structure in the PDOM for the control panel, in the "a11y view" it looked like this:
![capture](https://user-images.githubusercontent.com/6396244/44688149-50245080-aa21-11e8-9b43-ef285ae4e64d.PNG)

But I don't know if we want a full "Control Panel" now if it just contains a "Reset All" button. Ill ask at tomorrows a11y meeting.

If we do need that section back, Ill add it back with `scenery-phet/accessibility/AccessibleSectionNode` to remove the options duplication and hopefully make the purpose more clear. Leaving assigned to myself.

**Comment by jessegreenberg** [comment]
>  Ill add it back with ...
Same should be done for what is currently the `playAreaNode`.

Also,  related to https://github.com/phetsims/scenery-phet/issues/381 since these Nodes wouldn't need to exist at all in ScreenView subtypes and could be used with `accessibleOrder` in the future.

**Comment by jbphet** [comment]
I think most if not all sims have a reset button, so I'm a bit surprised that there isn't a standard approach for how to handle it when it's by itself.  Can the same approach be used as was done for, say, the friction sim?

**Comment by jessegreenberg** [comment]
Good point, a control section was requested in Friction so it is probably still necessary here. It would still be good to hear from @terracoda.

**Comment by jessegreenberg** [comment]
I went ahead and added the control area section back in.

**Comment by terracoda** [comment]
The removal of the Mute Sound button from the main area of the sim will affect several/many sims, and removing the Control Panel all together could affect content in the Scene Summary. We may need to discuss this issue a bit more and the general side affects of having a Control Area always or sometimes.

**Comment by emily-phet** [comment]
@terracoda My thought was that you might want to consider not having a "Control Area" for sims in the "simple" category - such as JT and Friction. It would only be sims whose description is structured in the 'simple' style. If this seems like an unnecessary complication, no need to worry about it. It also does not have to be decided now, we could consider this in the future, and if we decided to do it, apply it in the future to JT and Friction as they get new features.

**Comment by terracoda** [comment]
@emily-phet, this is what I was thinking, too, but thought it might be better to discuss with you first in a little more detail. I agree with the simple structure it does not add a lot of benefit to have another navigable region that has only a single control. 

I would like to see if there are other potential "simple sims" , like for example Ohm's Law and Resistance and a Wire. Those two sims have a bit more structure in the Play Area, but with the Mute Sound button in OL moved to the Sim Resources region, both sims' Control Areas would contain only the  Reset All button.

**Comment by jessegreenberg** [comment]
Discussed in 9/4/18 a11y meeting.

We should leave the sim as-is for now. In the future, we will consider whether or not simple sims like john-travoltage, and friction will require a "Control Area" section when it only includes very simple UI components like the ResetAllButton. An issue will be created in a11y-research for this.
